Source/Core/Core/IOS/USB/Emulated/Infinity.h

```diff
@@ -0,0 +1,124 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <array>
+#include <map>
+#include <mutex>
+#include <queue>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+#include "Common/IOFile.h"
+#include "Core/IOS/USB/Common.h"
+
+namespace IOS::HLE::USB
+{
+// Enum type, used when generating blank toys
+enum InfinityFigureType
+{
+  CHARACTER = 1,
+  PLAYSET = 2,
+  ABILITY_A = 3,
+  ABILITY_B = 4,
+  ITEM = 5
+};
+struct InfinityFigureInfo final
+{
+  InfinityFigureType type;
+  u16 figure_number;
+};
+
+extern const std::map<const char*, const InfinityFigureInfo> list_infinity_figures;
```

Comment:
is there a way you could make this not global? we don't really want to add more global/static state and in fact there are PRs to go the other direction and de-globalise things!

Reply:
I turned external access for this in to a public method, will likely do the same for the list os Skylanders eventually